### ISSUE TITLE
Please update alpine and latest tag to 2.48

### ISSUE THREAD

**Issue by ThStock** [jenkinsci/docker#442]
Please update `alpine` and  `latest` tag

```
$ docker pull jenkinsci/jenkins:alpine
alpine: Pulling from jenkinsci/jenkins
Digest: sha256:6d1f47fbefdaa898c2ac76d6c9c567353967a397fbc845d91aabff0709a74135
Status: Image is up to date for jenkinsci/jenkins:alpine
$ docker inspect --format='{{.RepoTags}}' jenkinsci/jenkins:alpine
[jenkinsci/jenkins:2.46-alpine jenkinsci/jenkins:alpine]
```
https://hub.docker.com/r/jenkinsci/jenkins/tags/
```
2.48-alpine 144 MB 2 days ago
2.48        312 MB 2 days ago
latest      312 MB 15 days ago
alpine      144 MB 16 days ago
```

**Comment by carlossg** [comment]
@rtyler do you have the logs to see what happened?

**Comment by rtyler** [comment]
```
Started by timer
Fetching changes from the remote Git repository
Checking out Revision 60bff698a9e096d75808400838782d45f06800c6 (refs/remotes/origin/master)
Loading library pipeline-library@master
Setting origin to https://github.com/jenkins-infra/pipeline-library.git
Fetching origin...
Fetching changes from the remote Git repository
Checking out Revision c4563c661fc9a80c06566b1e2765d11a5d4f8474 (master)
[Pipeline] properties
WARNING: The properties step will remove all JobPropertys currently configured in this job, either from the UI or from an earlier properties step.
This includes configuration for discarding old builds, parameters, concurrent builds and build triggers.
WARNING: Removing existing job property 'Discard old builds'
WARNING: Removing existing job property 'Build triggers'
[Pipeline] node
Running on trusted-agent-2 in /home/jenkins/trusted-workspace/workspace/Containers/jenkins
[Pipeline] {
[Pipeline] deleteDir
[Pipeline] stage
[Pipeline] { (Checkout)
[Pipeline] checkout
Cloning the remote Git repository
remote: Counting objects
remote: Compressing objects
Receiving objects
Resolving deltas
Updating references
Checking out Revision 60bff698a9e096d75808400838782d45f06800c6 (refs/remotes/origin/master)
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Publish)
[Pipeline] sh
[jenkins] Running shell script
+ ./publish.sh
Tag is already published: 2.32.2
Tag is already published: 2.33
Tag is already published: 2.34
Tag is already published: 2.35
Tag is already published: 2.36
Tag is already published: 2.37
Tag is already published: 2.38
Tag is already published: 2.39
Tag is already published: 2.40
Tag is already published: 2.41
Tag is already published: 2.42
Tag is already published: 2.43
Tag is already published: 2.44
Tag is already published: 2.45
Tag is already published: 2.46
Tag is already published: 2.47
Tag is already published: 2.48
Images 2.48 and latest are already the same, not updating tags
Images 2.32.2 and lts are already the same, not updating tags
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
Finished: SUCCESS
```

```
2.32.3-alpine: digest: sha256:42c77a578e16264a5d166bf931ccc9e51273e46a0f57c348c8a8184a8d47fa3a size: 3242
Tag is already published: 2.33-alpine
Tag is already published: 2.34-alpine
Tag is already published: 2.35-alpine
Tag is already published: 2.36-alpine
Tag is already published: 2.37-alpine
Tag is already published: 2.38-alpine
Tag is already published: 2.39-alpine
Tag is already published: 2.40-alpine
Tag is already published: 2.41-alpine
Tag is already published: 2.42-alpine
Tag is already published: 2.43-alpine
Tag is already published: 2.44-alpine
Tag is already published: 2.45-alpine
Tag is already published: 2.46-alpine
Tag is already published: 2.47-alpine
Tag is already published: 2.48-alpine
Images 2.48-alpine and alpine are already the same, not updating tags
curl: (22) The requested URL returned error: 404 Not Found
Images 2.32.3 and lts-alpine are already the same, not updating tags
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
Finished: SUCCESS
```

**Comment by carlossg** [comment]
ok, I've added some more logging, because it's running fine in my laptop, let's see next run

**Comment by rtyler** [comment]
@carlossg 

```
Tag is already published: 2.48
./publish.sh: line 84: tag1: unbound variable
[Pipeline] }
```
:frowning_face:

**Comment by frekele** [comment]
In Docker Hub, the tag jenkinsci/jenkins:latest is no longer being updated, more than 20 days without update.

**Comment by frekele** [comment]
In Docker Hub, the tag jenkinsci/jenkins:latest is no longer being updated, more than 20 days without update.
The last tag 2.49 was not released either. 😢

**Comment by carlossg** [comment]
@rtyler fixed that, let's check next run

**Comment by voorth** [comment]
job seems to have run, but still nothing on Docker Hub...

**Comment by carlossg** [comment]
fixed in dc869154621cc67d3a3b0decd9344b815ed89d07
